client/src/views/Settings/Tabs/EditPrivacy/index.vue

```diff
@@ -88,7 +90,18 @@ export default {
       like: true,
       reply: true,
       uploadPost: true,
-      hideOffencive: false
+      hideOffencive: false,
+      checked: []
+    }
+  },
+  methods: {
+    toggle () {
+      // this.$emit('input', !this.value)
+      // console.log(e.target)
+      // console.log(e.target.checked)
+      console.log(!this.value)
+      // this.checked.push(value)
+      console.log(this.checked)
```

Comment:
Uselesss method with comments and console?Why this here!?

Reply:
It should be in other pr. I pushed unfinished code here by mistake